README.md

```diff
@@ -49,6 +55,12 @@ are sporadic, then they will be pushed after the initial timeout window of 100 m
 (the spacer value) after the return of the previous handler, such that the network IO in the handler itself is given a bit
 of rest every time.
 
+### batch processing of data
+
+You listen on a stream that fires a lot of events with data attached. You would like to space out the processing of these events in time, 
+processing them in batches. Use the `Bufferer` to buffer these events and send out a full list of events to process in batch.
+Multiple event types are also supported as long as they have a common base class.
```

Comment:
It could be useful to mention here that it doesn't actually have to be "events" (in the sense of .NET events).
And if they *are* .NET events (or you want to restrict it to .NET events, even if they are of different types: according to https://learn.microsoft.com/en-us/dotnet/api/system.eventargs?view=net-8.0#remarks all .NET event data is supposed to be ultimately derived from EventArgs.

Reply:
Amended the README.